frontend/src/metabase/css/core/clearfix.module.css

```diff
@@ -10,20 +10,12 @@
  * 2. The use of `table` rather than `block` is only necessary if using
  *    `:before` to contain the top-margins of child elements.
  */
-:global(.clearfix::before),
-:global(.clearfix::after) {
+.clearfix::before,
+.clearfix::after {
   content: " "; /* 1 */
   display: table; /* 2 */
 }
 
-:global(.clearfix::after) {
+.clearfix::after {
   clear: both;
 }
-
-/**
- * For IE 6/7 only
- * Include this rule to trigger hasLayout and contain floats.
- */
-:global(.clearfix) {
```

Comment:
We even ended support for IE11, so I think this rule is pretty safe to be removed.